src/eg/src/guardian_share.rs

```diff
@@ -128,62 +155,99 @@ impl GuardianEncryptedShare {
         eg_hmac(&k0, &v)
     }
 
-    /// This function creates a new [`GuardianEncryptedShare`] of the dealer's secret key for a given recipient.
-    /// 
+    /// This function computes a [`GuardianEncryptedShare`] of a share for a given recipient and nonce.
+    ///
     /// The arguments are
```

Comment:
Arguments described in the documentation doesn't seem to be consistent with the code!

Reply:
Updated the comments to match the code.